libs/core/src/global/styles/tokens/base/_core.scss

```diff
@@ -143,6 +143,7 @@
   --pine-line-height-075: 0.9375;
   --pine-dimension-0: 0px;
   --pine-dimension-100: 8px;
+  --pine-dimension-125: 10px;
```

Comment:
Should components where 10px is hard coded be updated? Also, we'll need to ensure this gets updated in Token Studio.  (I think it may just be a pull in Token Studio once merged, but please confirm with @anechol )

Reply:
Tokens Studio is all good, we paired on getting that updated